src/index.js

```diff
@@ -0,0 +1,80 @@
+import './style.css';
+import pokemonCards from './modules/cards.js';
+import { addLike, getLikes } from './modules/likes.js';
+import openPopup from './modules/popup.js';
+import countItems from './modules/itemscounter.js';
+
+const BASE_API_URL = 'https://pokeapi.co/api/v2';
+const itemContainer = document.getElementById('itemContainer');
+
+// eslint-disable-next-line consistent-return
```

Comment:
- [ ] You're disabling too mush eslint rules, kindly make sure to follow and fix the linters instead of disabling them so your project follows the Microverse coding style